### DISCUSSION TITLE
[Issue] Payment of minus amount to customer account

### DISCUSSION THREAD

**Discussion by cryexx** [Blair2004/NexoPOS#1327] [Ideas]
We can not pay out minus amounts to the customer account. Voucher value higher than product value. How can we solve the problem?

**Expected behavior**
Voucher value is credited to the customer account - without detours

Voucher = 2€
Price for products = 1€
Checkout = -1€
-> Payout to customer account of 1€
-> payment amount: 0,00

Error message: Please enter a valid payment amount

**Screenshots**
![image](https://user-images.githubusercontent.com/88461700/215307320-519cedf4-f2f0-4649-b74d-48804f3ed355.png)

**Environment:**
 - NexoPOS 4.x vesrion: 4.8.13

**Comment by Blair2004** [comment]
You mean you want an option to store the change on the customer balance ?

**Comment by cryexx** [comment]
Yes, exactly. Both are best. To the customer account or cash.
